distro/configuration/encounterroles/encounterroles.csv

```diff
@@ -1,3 +1,4 @@
 Uuid,Void/Retire,Name,Description
 a0b03050-c99b-11e0-9572-0800200c9a66,,Unknown,Unknown encounter role for legacy providers with no encounter role set
-240b26f9-dd88-4172-823d-4a8bfeb7841f,,Clinician,"Doctor or Nurse who is the primary provider for an encounter, and will sign the note"
\ No newline at end of file
+240b26f9-dd88-4172-823d-4a8bfeb7841f,,Clinician,"Doctor or Nurse who is the primary provider for an encounter, and will sign the note"
+546cce2d-6d58-4097-ba92-206c1a2a0462,,Immunization Practitioner,Specialized clinician focused on administering vaccines
```

Comment:
I'd probably just change set the global property to point to `240b26f9-dd88-4172-823d-4a8bfeb7841f`.

Reply:
Nice one! updated.